docker/php-8.2/xdebug.ini

```diff
@@ -0,0 +1,14 @@
+;zend_extension = xdebug.so
+
+; XDebug 3 → https://xdebug.org/docs/upgrade_guide
+; You can dynamically enable XDebug by setting XDEBUG_MODE env variable.
+; Some options can be dynamically overridden with XDEBUG_CONFIG env variable.
+xdebug.mode = off
```

Comment:
it can be ambiguous, other versions of PHP have `xdebug.mode = debug`. I would keep it consistent

Reply:
I did it on purpose because IMHO debug should be opt-in. With `off` XDebug is transparent and extension can be enabled without performance overhead, but it shouldn't be set to `debug` by default.

But I agree consistency is good, so I would rather disable debug for other PHP versions, but in separate PR probably.